src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,17 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "a/12";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in this format: a/BLOCK, STREET, UNIT, POSTAL CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String ADDRESS_DELIMITER = ",";
+    
 
-    public final String value;
+    private final String value;
```

Comment:
do you still need to use have the value property since you have already stored the different parts of the address?  Try to avoid duplication to maintain consistency.